packages/ui-react/src/components/Player/Player.tsx

```diff
@@ -218,14 +214,8 @@ const Player: React.FC<Props> = ({
         playerOptions.autostart = autostart;
       }
 
-      // Set the license key if provided
-      if (playerLicenseKey) {
-        playerOptions.key = playerLicenseKey;
-      }
```

Comment:
Hi @mustran 👋 Is the analytics purpose the only reason for configuring a player license key? If we don't need a player license anymore, we probably should also clean up all the ini and dotenv configurations for this.

Reply:
Let's leave it "as is" for now. We will create a separate PR to handle player license key.